docs/migration/gcloud/vector.md

```diff
@@ -0,0 +1,162 @@
+# Vector
+
+*Migrating from Google Cloud to Siglens using Vector*
+
+## 1. Install Vector and setup Google Cloud CLI
+
+Prerequisites: 
+- Google Cloud CLI (Verify your Google Cloud CLI installation with `gcloud version` or you can install it from [here](https://cloud.google.com/sdk/docs/install-sdk))
+- Vector (You can follow [this guide](../../log-ingestion/vector.md#1-installation) to get started with installing vector)
+<details>
+<summary>Setup Google Cloud Pub/Sub</summary>
+
+#### 1. Set or Create a Google Cloud Project
+
+If you already have a Google Cloud project, set it as your current project:
+
+```bash
+gcloud config set project <project_id>
+```
+
+Replace `<project_id>` with your actual project ID.
+
+If you don't have a project, create one:
+
+```bash
+gcloud projects create <project_id>
+```
+
+#### 2. Create a Pub/Sub Topic
+
+Create a Pub/Sub topic where logs will be published:
+
+```bash
+gcloud pubsub topics create <topic_name>
+```
+
+#### 3. Create a Logging Sink
+
+When you create a sink, Google Cloud automatically creates a new service account. This service account, known as the logging service account, is used by the sink to write logs.
+
+```bash
+gcloud logging sinks create <sink_name> pubsub.googleapis.com/projects/<project_id>/topics/<topic_name>
+```
+#### 4. Grant Permissions to the Logging Service Account
+
+Grant the `pubsub.publisher` role to the logging service account:
+
+```bash
+gcloud projects add-iam-policy-binding <project_id> --member=serviceAccount:<logging_service_account> --role=roles/pubsub.publisher
+```
+
+Replace `<logging_service_account>` with the service account name displayed after creating the sink.
+
+#### 5. Create a Service Account
+> Skip this step if you have an existing service account
```

Comment:
Change the formatting on line 55. Right now it will show up as a quote reply, but you can make it appear as a Note by using the formatting tags we used here: https://github.com/siglens/siglens-docs/blob/develop/docs/log-ingestion/filebeat.md?plain=1#L8

Same comment for lines 66, 75, 83